README.md

```diff
@@ -154,3 +169,5 @@ Released under [MIT license][license]
 [waffle]: https://waffle.io/GlobalNamesArchitecture/gnparser
 [waffle_progress_svg]: https://badge.waffle.io/GlobalNamesArchitecture/gnparser.svg?label=in%20progress&title=In%20Progress
 [waffle]: https://waffle.io/GlobalNamesArchitecture/gnparser
+[jruby-installation]: http://jruby.org/getting-started
```

Comment:
I would keep references at the end, I think it  is cleaner, especially if the same reference is used many times